starter-code/index.js

```diff
@@ -1,17 +1,55 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = []
+    this.length = 0
 
-  add(item) {}
 
-  get(pos) {}
+  }
 
-  max() {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort(function (a, b) {
+      return a - b
+    });
+    this.length++
+  }
 
-  min() {}
+  get(pos) {
 
-  sum() {}
 
-  avg() {}
+    for (var i = 0; i < this.items.length; i++) {
```

Comment:
este for no haría falta porque si te fijas no lo estás recorriendo. Aun así acuérdate de poner let y no var